### PR TITLE
fix: 340 - updates to metrics collection to use modal ux

### PR DESCRIPTION
**Pull request by djmcquillan**
@SgtPooki closing this PR in preference of https://github.com/ipfs/public-gateway-checker/pull/373 after the formatting cleanup

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,30 +1,38 @@
 <!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (SgtPooki) [reply src/index.html:1]: https://github.com/ipfs/public-gateway-checker/pull/366

File: src/styles.css [hunk 2]
```diff
@@ -1,70 +1,72 @@
-body, html {
-	box-sizing: border-box;
-	text-align: center;
+body,
```
[the remark this answers is not in the document]
Comment (SgtPooki) [reply src/styles.css:1]: see https://github.com/ipfs/public-gateway-checker/pull/366.

**Review by SgtPooki — CHANGES_REQUESTED** [review]

**Comment by SgtPooki** [comment]
Testing the changes on the [preview build](https://bafybeidjcdcqe4yvke2j4qlcepomkwopjf2h3lihkfeflefuixdhj2m4vm.on.fleek.co/) look good except for one thing:

Metrics are disabled upon page load:

image

we could probably address this issue and a few of my comments on this PR in another issue, https://github.com/ipfs/public-gateway-checker/issues/367

**Comment by tinytb** [comment]
Replaced by https://github.com/ipfs/public-gateway-checker/pull/373

**Pull request closed without merging**